sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricsExtractionProcessor.cs

```diff
@@ -41,44 +35,37 @@ public override void OnEnd(Activity activity)
                 _manager.LiveMetricsResource = LiveMetricsResource;
             }
 
-            string? statusCodeAttributeValue = null;
-
-            foreach (ref readonly var tag in activity.EnumerateTagObjects())
-            {
-                if (tag.Key == SemanticConventions.AttributeHttpResponseStatusCode)
-                {
-                    statusCodeAttributeValue = tag.Value?.ToString();
-                    break;
-                }
-            }
-
             if (activity.Kind == ActivityKind.Server || activity.Kind == ActivityKind.Consumer)
             {
-                AddRequestDocument(activity, statusCodeAttributeValue);
+                AddRequestDocument(activity);
             }
             else
             {
-                AddRemoteDependencyDocument(activity, statusCodeAttributeValue);
+                AddRemoteDependencyDocument(activity);
             }
 
             if (activity.Events != null)
             {
                 foreach (ref readonly var @event in activity.EnumerateEvents())
                 {
-                    string? exceptionType = null;
-                    string? exceptionMessage = null;
+                    string exceptionType = string.Empty;
+                    string exceptionMessage = string.Empty;
```

Comment:
Need to first check the event name here to confirm it is an exception event.

Reply:
abandoned this implementation. Switched to ActivityTagsProcessor from Exporter